src/tests/index.test.js

```diff
@@ -1,24 +1,59 @@
-import { notificationsChannelURLFromHookURL } from '../index';
+import { notificationsChannelURLFromHookURL } from '../utils';
 
 describe('notificationsChannelURLFromHookURL tests', () => {
-  test('New domains', () => {
-    // localhost
+  test('Old domains tests', () => {
+    // production
+    expect(
+      notificationsChannelURLFromHookURL(
+        'https://api.production.twyla.io/widget-hook/massive-dynamic/z10'
+      )
+    ).toEqual({
+      botName: 'Z10',
+      notificationsChannelURL:
+        'wss://notification.production.twyla.io/widget-notifications/massive-dynamic/z10',
+      projectName: 'z10',
+      workspaceName: 'massive-dynamic',
+    });
+
+    // demo
     expect(
-      notificationsChannelURLFromHookURL('http://127.0.0.1:8080/widget-hook/massive-dynamic/z10')
-    ).toEqual('ws://127.0.0.1:8080/widget-notifications/massive-dynamic/z10');
+      notificationsChannelURLFromHookURL(
+        'https://api.demo.twyla.io/widget-hook/massive-dynamic/z10'
+      )
+    ).toEqual({
+      botName: 'Z10',
+      notificationsChannelURL:
+        'wss://notification.demo.twyla.io/widget-notifications/massive-dynamic/z10',
+      projectName: 'z10',
+      workspaceName: 'massive-dynamic',
+    });
+  });
 
+  test('New domains', () => {
     // rawhide
     expect(
       notificationsChannelURLFromHookURL(
         'https://api.rawhide.canvas.twyla.ai/widget-hook/massive-dynamic/z10'
       )
-    ).toEqual('wss://api.rawhide.canvas.twyla.ai/widget-notifications/massive-dynamic/z10');
+    ).toEqual({
+      botName: 'Z10',
+      notificationsChannelURL:
+        'wss://notification.rawhide.canvas.twyla.ai/widget-notifications/massive-dynamic/z10',
```

Comment:
This is a regression. Notification endpoint is deprecated.

Reply:
removed